test/tests/auxkernels/current_boundary_id/current_boundary_id.i

```diff
@@ -0,0 +1,75 @@
+#
+# This is used to create the mesh but it does not work with --distributed-mesh flag
+# and the parallel test bombs.
```

Comment:
The testing showed that this really fails with distributed mesh, try adding `parallel_type = replicated` into the `Mesh` block. That should prevent this test to run with distributed mesh.

I am assuming the distributed mesh restriction is caused by the `MeshGeneratorMesh` class? May be you wanted to use the `current_boundary_id_in.e` file? It is checked in your PR...

Reply:
I meant to change this back to the commented `FileMesh` block and forgot.